### ISSUE TITLE
Vehicle capacity on build vehicle pages missing or corrupted when distances and speeds set to Metric

### ISSUE THREAD

**Issue by shusaura85** [OpenLoco/OpenLoco#1557]
**Version information:**
 - Platform: Windows 10
 - Version: 22.06.1 (release) using GOG version of Locomotion

**Describe the bug**
Vehicle capacity description when buying new vehicles is no longer visible or shows garbage text. This worked in 22.05.1.  

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to any of the 'Build trains / buses / trucks / trams / aircraft / ships' menus
2. Hover on any of the vehicles in the purchase list
3. In the right side of the window, it should show vehicle capacity
4. Vehicle capacity is either missing or shows garbage text

**Expected behaviour**
The actual vehicle capacity should be shown.

**Screenshots**
![ss1](https://i.imgur.com/s6O6Dhn.png)
![ss1](https://i.imgur.com/3QiKN9x.png)
![ss1](https://i.imgur.com/QZAlbD2.png)
![ss1](https://i.imgur.com/XEuji1s.png)

**Additional context**
No mods or other changes to game files were made, just simple upgrade from 22.05.1 to 22.06.1  
  
Problem was most likely introduced in pull request #1550

**Comment by shusaura85** [comment]
Removing the files created in %appdata%\openloco (essentially making it a new install) apparently fixed the issue.  
Will try restoring the files one by one to see what file created the issue.

**Comment by shusaura85** [comment]
I tried restoring the original files, when I restored the game.cfg in %appdata%\openloco the problem appeared again.  
After that, I tried resettting the configuration options and I found the option that causes this problem.  
  
Options - Regional - Distance and Speed. This is the option in question.
 - If the setting is set to *Imperial* it shows the capacity without problem.  
 - If the setting is set to *Metric* it won't show the capacity correctly.


![imperial](https://i.imgur.com/k1b6S9m.png)
![metric](https://i.imgur.com/9fw6xgI.png)
